webserver/exceptions.py

```diff
@@ -0,0 +1,13 @@
+class InvalidAPIUsage(Exception):
```

Comment:
Maybe it would be better to have more generic `APIException` and use a separate exception in cases where we have no data? Something like this:

```python
class APIException(Exception): pass
class BadRequest(APIException): pass
class NoData(APIException): pass
# etc...
```

And just set status associated status code in each instead of setting it in constructor every time. What do you think?